lib/hooks/auth/useLogin.ts

```diff
@@ -37,24 +37,36 @@ export const useLogin = () => {
         timeout: process.env.NODE_ENV === "production" ? 60000 : 0,
       });
 
-      if (data?.error) {
-        const error = data.error;
-        if (hasError("UserNotConfirmedException", error)) {
+      // TODO: are there other challengeState/states to check for? It looks like the rest are errors (API responses)
```

Comment:
@craigzour from what I can see in [...nextauth].ts this is the only success message. Would you know if this is right?

Reply:
And yes the `challengeState` can only have the `MFA` value associated to it for now.

Reply:
Got it - thanks!